code/datums/shuttles.dm

```diff
@@ -538,6 +538,12 @@
 	name = "science outpost shuttle"
 	can_be_bought = FALSE
 
+/datum/map_template/shuttle/science/aus
+	port_id = "science"
+	suffix = "outpost_aus"
+	name = "science outpost shuttle"
+	can_be_bought = FALSE
```

Comment:
just modify the suffix var on the original one, map edits are so so much worse than just changing code